### PR TITLE
Support importing css and css modules from specs

### LINKED ISSUE #51: Css loaders
It would be great, if we could use a css loader or css modules.
Is this planned?
tks!!

### PR DESCRIPTION
**Pull request by frederickfogerty**
I thought this had been sitting around for a while, so I decided to fix it.

----

This is an extension of #52. I opened a new PR as I had to rebase and didn't want to force push.

This builds on work done by @philcockfield, all I did was fix a bug.

Basically, as we wanted to support normal css files **and** css modules at the same time, webpack was matching the CSS Module files to the CSS Modules loader *and then to the normal css loader* causing the CSS to be parsed twice. The second parsing was failing as the CSS was now JS, and could not be parsed as CSS (understandably). 

Simply, I just stopped the module files being parsed by the normal css loader using some regex hacking. For more information, look at the (extensive) comment in webpack.config.js, around L155.

Fixes #51 

🎉

### CODE REVIEW

File: src/specs/css-loader.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { lorem } from './util';
+
+
+const IS_BROWSER = typeof(window) !== 'undefined';
+if (IS_BROWSER) {
+  require('./css-loader.css');
+}
```
Comment (frederickfogerty) [inline src/specs/css-loader.spec.jsx]: @philcockfield if we no longer load the js files on the server, do we need these hacks?

File: src/server/webpack-config.js [hunk 2]
```diff
@@ -124,6 +127,74 @@ export default (options = {}) => {
...
+  const { loaders } = config.module;
+  const { cssModules } = options;
+  const simpleCssLoader = { test: /\.css$/, loader: 'style!css' };
+  if (cssModules) {
+    let simpleLoaderAdded = false;
+    cssModules.forEach(test => {
+      loaders.push({
+        test,
+        loader: 'style!css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', // eslint-disable-line max-len
+        exclude: /node_modules/,
```
Comment (frederickfogerty) [inline src/server/webpack-config.js]: @philcockfield can you make sure this line doesn't break the ui-harness when it's used in other projects - i.e. css modules are still loaded correctly from npm packages... I have a sneaking suspicion they won't be

Comment (philcockfield) [inline src/server/webpack-config.js]: Will do!

**Comment by philcockfield** [comment]
@frederickfogerty - Duuuude! 😎

**Comment by philcockfield** [comment]
Oh - I thought this had been merged already! ( @frederickfogerty )

**Comment by philcockfield** [comment]
I'll get this synched with master and merged today @frederickfogerty

**Comment by klavs** [comment]
Is today over?

**Comment by philcockfield** [comment]
image

**Comment by philcockfield** [comment]
The day is finally done @klavs!  Thanks @frederickfogerty for all your hard work.

**Comment by philcockfield** [comment]
Released as `3.14.0`.

**Pull request merged**